pydis_site/apps/api/models/bot/nomination.py

```diff
@@ -52,3 +45,33 @@ class Meta:
         """Set the ordering of nominations to most recent first."""
 
         ordering = ("-inserted_at",)
+
+
+class NominationEntry(ModelReprMixin, models.Model):
+    """A nomination entry created by single staff."""
+
+    nomination = models.ForeignKey(
+        Nomination,
+        on_delete=models.CASCADE,
+        help_text="Nomination to what this entry belongs."
+    )
+    actor = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        help_text="The staff member that nominated this user.",
+        related_name='nomination_set'
+    )
+    reason = models.TextField(
+        help_text="Why the actor nominated this user.",
+        null=True,
+        blank=True
```

Comment:
Do we want two different options to represent an empty reason, `""` and `null`? Could you explain why? The Django docs recommend you to not use `null=True` for character-based fields, as they already have a "logical" null value being an empty string.